config/nmslib/tool.yml

```diff
@@ -1,8 +1,8 @@
 test_name: nmslib_test
-test_type: index
+test_id: 3
```

Comment:
Hmmm I think I rather have each test have a unique descriptive name here as opposed to an id.

Reply:
I've changed `test_id` to be a descriptive string instead